2_process/src/process_and_style.R

```diff
@@ -1,19 +1,31 @@
-process_data <- function(nwis_data){
-  nwis_data_clean <- rename(nwis_data, water_temperature = X_00010_00000) %>% 
-    select(-agency_cd, -X_00010_00000_cd, tz_cd)
+combine_nwis_data <- function(download_files){
+
+  for (download_file in download_files){
+
+    # read the downloaded data and append it to the existing data.frame
+    these_data <- read_csv(download_file, col_types = 'ccTdcc')
+    data_out <- rbind(data_out, these_data)
```

Comment:
I think `data_out` is not available to be included in the `rbind()` for the first file being bound.